tests/Login_unit_tests.cpp

```diff
@@ -0,0 +1,16 @@
+#include <gtest/gtest.h>
+
+#include "../include/Login.hpp"
+
+namespace {
+
+TEST(Login, createUser) {
+    /* Test creation of users with valid and invalid input. */
+    Login login = Login();
+    EXPECT_EQ(login.createUser("Jon", "12345", "owner"), true);
+    EXPECT_EQ(login.createUser("Jon", "12345", "Baconator"), false);
```

Comment:
Where's the beef?

Reply:
I know I know, I didn't get my beef in there... I apologize and we can fix it in a future update.